### DISCUSSION TITLE
Any forum of zrepl's users?

### DISCUSSION THREAD

**Discussion by mtopczewski** [zrepl/zrepl#597] [Q&A] [answered]
Hi, I'm newbe as it's easy to quess ... :-) I have problem with run cases from "Quick start by Use Case" from offical tutorial. I suppose I'm doing some basic mistakes, so i'm looking for web forum of zrepl users where I could ask some of trivial questions...
Best regards,
Mariusz

**Comment by gdevenyi** [comment]
Technically, you're using it.

**Comment by problame** [comment, marked as answer]
Yeah, just post into the Q&A section! Right on this site!

**Comment by mtopczewski** [comment]
Thanks, I will post then the question in separate thread.
